### PR TITLE
Updates/table gen

### PR DESCRIPTION
**Pull request by rjullman**

### CODE REVIEW

File: atlasdb-client/src/main/java/com/palantir/atlasdb/schema/generated/SweepPriorityTable.java [hunk 1]
```diff
@@ -155,7 +161,7 @@ public String apply(SweepPriorityRow row) {
             return new Function<String, SweepPriorityRow>() {
                 @Override
                 public SweepPriorityRow apply(String row) {
-                    return new SweepPriorityRow(row);
+                    return SweepPriorityRow.of(row);
```
Comment (markelliot) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/schema/generated/SweepPriorityTable.java]: generated code for this line and for line 181 are out of sync

File: atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/RowOrDynamicColumnRenderer.java [hunk 2]
```diff
@@ -186,15 +171,22 @@ private void bytesHydrator() {
                     }
                     line("__index += ", comp.getType().getHydrateSizeCode(var), ";");
                 }
-                line("return of(", Joiner.on(", ").join(vars), ");");
+                line("return new ", Name, "(", Joiner.on(", ").join(vars), ");");
```
Comment (markelliot) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/RowOrDynamicColumnRenderer.java]: per above, I guess this should be

    line("return ", Name, ".of(", Joiner.on(", ").join(vars), ");");

**Comment by markelliot** [comment]
might consider squashing commits before merging

**Comment by clockfort** [comment]
+2 lgtm

**Pull request merged**